flows/example-test.js

```diff
@@ -5,23 +5,19 @@ class exampleUserFlow {
     // return new Promise(async (resolve, reject) => {
     const page = await browser.newPage();
```

Comment:
I'm curious, how will this flow file be used? Is this a sample that you're using for testing? Will end users have to write their own, and if so, how does it get injected into the racer?